### PR TITLE
Enable to set custom GitHub url

### PR DESCRIPTION
**Pull request by mataku**
Hi, thanks for your amazing library.

I want to use this library in the project managed at GitHub Enterprise. So, enabled to set custom GitHub URL via property.

To connect to GitHub, this library uses https://github.com/maxirosson/jdroid-java-github which I thought it could use custom GitHub URL. So, changed only in this library.

If not, I will send pull-request to [https://github.com/maxirosson/jdroid-java-github](https://github.com/maxirosson/jdroid-java-github)!

### CODE REVIEW

**Comment by maxirosson** [comment]
Hi. Thanks for your collaboration !!!

Maybe we should use the `GitHubClient` constructor that receives the hostName, and change the prop to `gitHubApiHostName`

**Comment by mataku** [comment]
@maxirosson Thanks!

I will fix it and try in my project.

File: releases-hub-gradle-plugin/src/main/java/com/releaseshub/gradle/plugin/ReleasesHubGradlePlugin.kt [hunk 1]
```diff
@@ -52,6 +52,7 @@ class ReleasesHubGradlePlugin : Plugin<Project> {
             upgradeDependenciesTask.gitHubRepositoryOwner = extension.gitHubRepositoryOwner
             upgradeDependenciesTask.gitHubRepositoryName = extension.gitHubRepositoryName
             upgradeDependenciesTask.gitHubWriteToken = extension.gitHubWriteToken
+            upgradeDependenciesTask.gitHubApiUrl = extension.gitHubApiHostName
```
Comment (maxirosson) [inline releases-hub-gradle-plugin/src/main/java/com/releaseshub/gradle/plugin/ReleasesHubGradlePlugin.kt]: The prop name on the task should be `gitHubApiHostName`

Comment (mataku) [reply releases-hub-gradle-plugin/src/main/java/com/releaseshub/gradle/plugin/ReleasesHubGradlePlugin.kt]: Sorry, fixed!

**Pull request merged**